admin/mac/macdeploy.py

```diff
@@ -229,11 +229,11 @@
 ]
 
 QT_PLUGINS_SEARCH_PATH=[
-    '/usr/local/Cellar/qt5/5.4.0/plugins',
+    '/usr/local/opt/qt5/plugins',
 ]
 
 SNORE_PLUGINS_SEARCH_PATH=[
-    '/usr/local/Cellar/snorenotify/HEAD/lib/libsnore',
+    '/usr/local/opt/snorenotify/lib/plugins/libsnore-qt5',
```

Comment:
Nope, this is brew. 
Maybe this is new but Homebrew always links the installed version from $BREW_PREFIX/opt/$FORMULA_NAME/